### DISCUSSION TITLE
Issue resolved: "Could not find game data"

### DISCUSSION THREAD

**Discussion by qreeves** [redeclipse/base#1173] [General]
The recent name change on GitHub seems to have triggered an issue with our auto-updater which fell through the cracks on my local system. I've managed to work around the issue, but if you get "`Could not find game data`", please just run the game again, the update will fix the updater the first time and should work properly on subsequent attempts.

**Comment by cloudziq** [comment]
I can confirm that it works now, but I needed to uncheck 'read-only' attribute on /data folder

**Comment by oliver-ruehl** [comment]
Hello, I'm a regular user and gamer. It is not working for me at all with Version 1.6.0
I can't fix it by restarting the game with Windows 10.
Removing the read-only attribute does not work as well.
I also tried to install it for various users and changed permissions etc.

**Comment by no-lex** [comment]
Can you provide a log of the output from the updater script, `redeclipse.bat` ?

**Comment by oliver-ruehl** [comment]
Checking for updates to "stable". To disable: set REDECLIPSE_BRANCH=inplace

branch: stable
folder: D:\Spiele\redeclipse_1.6.0_win\redeclipse-1.6.0-win
cached: C:\Users\Admin\Documents\My Games\Red Eclipse\cache\stable
modules: Updating..
base: b58c9d0f08b241459625512c2ef92e94c4ff8d75 is installed.
base: b58c9d0f08b241459625512c2ef92e94c4ff8d75 is the current version.
base: Not updated, skipping other modules...
bins: Updating..
bins: b58c9d0f08b241459625512c2ef92e94c4ff8d75 is installed.
bins: b58c9d0f08b241459625512c2ef92e94c4ff8d75 is the current version.

**Comment by cloudziq** [comment]
make sure permisions for these folders are set to read and write:
RE install folder/data
documents/games/redeclipse/cache

then delete version.txt from RE install folder and the same file in /bin folder
then run redeclipse.bat again :)

**Comment by oliver-ruehl** [comment]
That was clunky, but it worked :)
Thanks for that. My kid is really happy now we can play again together hehe.

Will there be a fix that'll make it a bit mor snazzy next time? :)
Thanks for your work.

**Comment by qreeves** [comment]
It's a one time fix to address an issue with renaming our GitHub organisation and only affects people who tried to update while it was broken. I'll probably be ditching the hacky updater in favour of getting people to use the [GameJolt client](https://gamejolt.com/client) or [Steam](https://store.steampowered.com/about/) in [v2.0](https://www.redeclipse.net/docs/Information-for-v2).

**Comment by oliver-ruehl** [comment]
I'll be using the Steam version then when its out.
Thanks 🥇
